airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -214,5 +217,5 @@ class ClickView(IncrementalGoogleAdsStream):
     ClickView stream: https://developers.google.com/google-ads/api/reference/rpc/v8/ClickView
     """
 
-    time_unit = "days"
     days_of_data_storage = 90
+    range_days = 1
```

Comment:
And description of this overriding of the value coul be usefel as well.

Reply:
Before the changes provided in this PR, `time_unit` was overriden with "days" and 
`start_date = start_date.add(**{time_unit: 1})`, the final result means `start_date.add(days=1)`
So, to save the behaviour, just set `range_days` to 1,